plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestParquetWriterConfig.java

```diff
@@ -60,13 +61,15 @@ public void testExplicitPropertyMappings()
                 "parquet.experimental-optimized-writer.enabled", "true",
                 "parquet.writer.block-size", "234MB",
                 "parquet.writer.page-size", "11MB",
-                "parquet.writer.batch-size", "100");
+                "parquet.writer.batch-size", "100",
+                "parquet.optimized-writer.validation-percentage", "10");
```

Comment:
Why 10? Shouldn't we check all data in tests? Or is it too expensive?

Reply:
This is just testing of non-default value in config.
Validation is enabled for tests using ParquetTester and in TestHiveFileFormats#testOptimizedParquetWriter.
I had also run CI with validation default set to 100% and verified that it was green.